xeofs/models/eeof.py

```diff
@@ -105,12 +110,13 @@ def __init__(
                 compute=self._params["compute"],
                 sample_name=self.sample_name,
                 feature_name=self.feature_name,
+                solver_kwargs=self._params["solver_kwargs"],
```

Comment:
We probably want to incorporate the advantage of the optional sanity check here in the EEOF model. By that point, the data has already been processed by the Preprocessor, making the `NaN` check redundant.

EDIT: Not optional, but actually setting `check_nans=False`

```suggestion
                compute=self._params["compute"],
                check_nans=False,
                sample_name=self.sample_name,
                feature_name=self.feature_name,
                solver_kwargs=self._params["solver_kwargs"],
```